catkin_tools/jobs/catkin.py

```diff
@@ -102,6 +130,7 @@ def create_catkin_build_job(context, package, package_path, dependencies, force_
         'make',
         [MAKE_EXEC] + make_args,
         cwd=build_space,
+        env=ctr_env if 'test' in make_args else {},
```

Comment:
Should the `ctr_env` extend the current env rather than replace it? Something like this:

```python
new_env = dict(os.environ)
new_env.update(ctr_env if 'test' in make_args else {})
# ...
        env=new_env,
```